pkg/api/handlers/types.go

```diff
@@ -428,6 +444,18 @@ func LibpodToContainerJSON(l *libpod.Container) (*docker.ContainerJSON, error) {
 		return nil, err
 	}
 
+	if sz {
+		if sizeRW, err = l.RWSize(); err != nil {
+			return nil, err
+		}
+		if sizeRootFs, err = l.RootFsSize(); err != nil {
+			return nil, err
+		}
+	} else {
```

Comment:
No need for else block, These are defaults.

Reply:
The entire if sz block was wrong, have removed and changed the call to inspect to pass sz rather than it being hard coded to true, which seemed a better thing to do